### DISCUSSION TITLE
homeproxy能不代理路由器吗

### DISCUSSION THREAD

**Discussion by ragingbulld** [immortalwrt/homeproxy#173] [General]
用了homeproxy就不能使用natmap了

**Comment by muink** [comment]
natmap 里面设定 fwmark 为 100 即可
目前只有 natmapt 这个 openwrt 上的实现支持这个功能
<img width="815" alt="image" src="https://github.com/user-attachments/assets/187297d4-be0b-4433-8a1d-77adb8d26f32">

**Comment by 1715173329** [comment]
可以自行去除相关规则。
